man/coap_session.txt.in

```diff
@@ -303,13 +319,40 @@ setup_client_session_pki (struct in_addr ip_address,
 ----
 #include <coap@LIBCOAP_API_VERSION@/coap.h>
 
+#include <stdio.h>
 #include <netinet/in.h>
 
+#ifndef min
+#define min(a,b) ((a) < (b) ? (a) : (b))
+#endif
+
+static const coap_dtls_cpsk_info_t *
+verify_ih_callback(coap_str_const_t *hint,
+                   coap_session_t *c_session,
+                   void *arg
+) {
+  coap_dtls_cpsk_info_t *psk_info = (coap_dtls_cpsk_info_t *)arg;
+  char lhint[128];
+  /* Remove (void) definition if variable is used */
+  (void)c_session;
+
+  snprintf(lhint, sizeof(lhint), "%.*s", (int)hint->length, hint->s);
```

Comment:
Pass `"%.*s"` to `coap_log()`, see above.

Reply:
Updated